src/dashboard/src/utils/convertToStorageSize.ts

```diff
@@ -3,15 +3,21 @@
  * @param value The initial value to convert.
  * @param input The input size type.
  * @param output The output size type.
+ * @param options Configuration options.
  * @returns A string representing the storage size.
  */
-export const convertToStorageSize = (
+export const convertToStorageSize = <T extends string | number>(
```

Comment:
Needed a way to return just the converted value.  I don't love the generics in this case.